addons/packages/contour/1.22.0/bundle/config/overlays/update-envoy-daemonset.yaml

```diff
@@ -4,9 +4,34 @@
 
 #@overlay/match by=overlay.subset({"kind": "DaemonSet", "metadata": {"name": "envoy"}})
 ---
+kind: #@ data.values.envoy.workload.type
```

Comment:
Would be good to add some validation on this field in https://github.com/vmware-tanzu/community-edition/blob/main/addons/packages/contour/1.22.0/bundle/config/contour.star#L117, to only allow `Deployment` or `DaemonSet`. The overlay looks safe but still better UX to explicitly validate rather than ignore invalid values.

Reply:
ah good point, I added some now. What do you think?

Reply:
:+1: lgtm